src/applications/personalization/dashboard-2/components/claims-and-appeals/Appeal.jsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import moment from 'moment';
+
+import {
+  APPEAL_TYPES,
+  EVENT_TYPES,
+  getTypeName,
+  getStatusContents,
```

Comment:
I looked at what you're importing quite thoroughly since we're looking to not get logic mixed with other applications, and I agree that importing this stuff directly from the claims-status tool makes sense here. 👍

Reply:
Plus we already were importing a lot of stuff from claims and appeals.